apps/web/src/components/GetStarted/BuildersMostWanted.tsx

```diff
@@ -0,0 +1,91 @@
+import classNames from 'classnames';
+import { sectionContainerClasses, gridClasses } from '../../../app/(base-org)/getstarted/page';
+import { Icon } from '../Icon/Icon';
+import { Resource } from './resourceTypes';
+import ResourceCard from './ResourceGrid/ResourceCard';
+import { FUNDING_SECTION_ID } from './Funding';
+import { GET_NOTICED_SECTION_ID } from './GetNoticed';
+import { START_BUILDING_SECTION_ID } from './StartBuilding';
+
+type MostWantedResource = Resource & {
+  icon: string;
+};
+
+const mostWanted: MostWantedResource[] = [
+  {
+    title: 'Create your profile',
+    description: 'Claim a Basename and create your Based Profile to connect with other  Builders',
+    href: '/names?utm_source=dotorg&utm_medium=builderkit',
+    icon: 'basenamesIcon',
+  },
+  {
+    title: 'Office Hours',
+    description: 'Schedule time to talk to directly to a member of the Base Ecosystem team',
+    href: 'https://lu.ma/base-officehours/?utm_source=dotorg&medium=builderkit',
+    icon: 'dotGrid',
+  },
+  {
+    title: 'Host a Meetup',
+    description: 'Sign up to host a meetup with other Based builders anywhere in the world',
+    href: '/why-base/?utm_source=dotorg&utm_medium=builderkit',
+    icon: 'questionMark',
+  },
+  {
+    title: 'Get Funded',
+    description: 'A collection of monetary programs to help you build or grow your project',
+    href: `#${FUNDING_SECTION_ID}`,
+    icon: 'cash',
+  },
+  {
+    title: 'Get Noticed',
+    description:
+      'Looking for help with distribution? Get noticed by millions of potential new users',
+    href: `#${GET_NOTICED_SECTION_ID}`,
+    icon: 'people',
+  },
+  {
+    title: 'Build Your Project',
+    description: 'Resources that make it easy to build and use your onchain project',
+    href: `#${START_BUILDING_SECTION_ID}`,
+    icon: 'builderHammer',
+  },
+];
+
+export default async function BuildersMostWanted() {
+  const titleClasses = classNames(
```

Comment:
These can live outside the render and don't need classNames, yeah?

Reply:
yep, moved outside the render and made all classnames strings into template strings